app/controllers/locations_controller.rb

```diff
@@ -0,0 +1,5 @@
+class LocationsController < ApplicationController
+  def index
+    @locations = Locations.near_postcode(params['postcode'])
```

Comment:
As we discussed when writing the gherkins, we have no concept of what is 'near' just what it is 'nearest'. I think the API for `Location` should reveal this.

```rb
Locations.nearest_to_postcode
```